pkg/api/v1/atlasbackupcompliancepolicy_types_test.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+import "testing"
+
+func TestBackupCompliancePolicyToAtlas(t *testing.T) {
+	t.Run("Can convert Compliance Policy to Atlas", func(t *testing.T) {
+
```

Comment:
let's add some juice here :-)

Reply:
+1

Reply:
Added some tests here, let me know if you think this needs more thorough testing

Reply:
commented elsewhere, we can improve coverage, resolving this comment for now.